test/io-tests/configs/secret-from-file.config

```diff
@@ -0,0 +1,10 @@
+db-uri = "postgres:///postgrest_test"
+db-schema = "test"
+db-anon-role = "postgrest_test_anonymous"
+db-pool = 1
+server-host = "*4"
+server-port = 49421
+
+# Read secret from a file: /dev/stdin (alias for standard input)
+jwt-secret = "@/dev/stdin"
```

Comment:
`-` is a convention which has to be implemented by the software (hint: PostgREST doesn't).
`/dev/stdin` is a symbolic link which points to the standard input of the current process; this is an OS feature.